PyTDF/backend/Local.py

```diff
@@ -0,0 +1,17 @@
+import ROOT
+
+def local_executor(generator):
+    """
+    Execution of the event-loop
+    in local environment
+
+    """
+    mapper = generator.get_mapper_callable()
+    generator.root_node.create_tdf()
```

Comment:
Here you will create the real TDF in place, without going to the generator.

Reply:
This is only until the new version of ROOT is released. In the older version (with TDataFrame instead of RDataFrame), the head node of TDF will have to be kept around for execution of event loop and for `GetValue()` execution with TResultPtr

Reply:
the requirement is still there in v6.14 (see my comment above).
but the requirement that the head node needs to stay in scope is unrelated to where you create it, right?

Reply:
Yes. That's true. If that's the case, I can manually create the TDF head node here and then assign it to a member in PyTDF object. That'd work right ?

Reply:
> assign it to a member in PyTDF object

you shouldn't need to: the TDF head node only needs to be alive when running the event loop, which happens inside the backend.
and anyway in the distributed case you could not do it, since the TDF head node would be constructed in the workers, which have no access to the original pyTDF graph.